backend/vouch/models.py

```diff
@@ -0,0 +1,69 @@
+from django.db import models
+from django.db.models import ObjectDoesNotExist
+from core.models.user import User
+
+# Create your models here.
+class vouch(models.Model):
+
+    vouching = models.ForeignKey(
+        User,
+        on_delete=models.CASCADE,
+        help_text="",
+        related_name="vouchinguser",
+    )
+    vouchedfor = models.ForeignKey(
+        User,
+        on_delete=models.CASCADE,
+        help_text="The contributor",
+        related_name="vouchedforuser",
+    )
+    is_public = models.BooleanField(
+        default=False, null=False, help_text="Should the vouching be public?"
+    )
+
+    trust_value = models.FloatField(
+        default=0,
+        blank=False,
+        help_text="trust value assigned by vouchinguser to vouchedforuser",
+    )
+    uncertainty = models.FloatField(
+        default=0,
+        blank=False,
+        help_text="Uncertainty about the vouching score",
+    )
+    criteria = models.TextField(
+        default=0,
```

Comment:
Hi, actually I put this field but don't necessarily intend to use it directly, the idea was to be able to store for which aspect the user vouching has vouched for (for example one could say "I vouch for this user having this diploma, but I don't know anything about their competencies"), but I thought it would be easier to start by just allowing users to vouch for a user in their entirety, and then improve it if time allows. May I ask you for a suggestion on how you would do that properly?